### PR TITLE
chore: merge master in release/2.129.0

### PR DESCRIPTION
**Pull request by theodu**

### CODE REVIEW

**Comment by Jonas1312** [comment]
we need to merge this one to fix the pyright errors: https://github.com/kili-technology/kili-python-sdk/pull/1047

File: setup.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.129.3
```
Comment (Jonas1312) [inline setup.cfg:2]: we will have to double bump before next release

File: src/kili/__init__.py [hunk 2]
```diff
@@ -5,4 +5,4 @@
 Kili Python SDK
 """
 
-__version__ = "2.129.3"
```
Comment (Jonas1312) [inline src/kili/__init__.py:8]: same here

**Review by Jonas1312 — APPROVED** [review]
just need to bump

File: setup.cfg (hunk 1 above)
Comment (theodu) [reply setup.cfg]: I just added the commit of bump to restore 2.129.3 on the branch

**Pull request merged**